src/stcal/ramp_fitting/ramp_fit.py

```diff
@@ -274,4 +274,5 @@ def suppress_one_good_group_ramps(ramp_data):
             # ramp saturated, mark the 0th groups as saturated, too.
             good_index = np.where(ramp_data.groupdq[integ, :, row, col] == 0)
             if ramp_data.groupdq[integ, good_index, row, col] == 0:
-                ramp_data.groupdq[integ, good_index, row, col] = dnu_flag
+                ramp_data.groupdq[integ, :, row, col] = np.bitwise_or(
```

Comment:
Line 267 - 'flagg'; and should that comment (and the comment that follows that) be changed to state that ALL groups will be suppressed ?

Reply:
I fixed the comments in this section to reflect the new change in ramp suppression.